fishtank-cli/src/commands/spawn.ts

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Command, Config } from '@oclif/core'
+import { Flags } from '@oclif/core'
+import { Cluster } from 'fishtank'
+
+export abstract class Spawn extends Command {
+  static description = 'Spin up a new node inside a cluster'
+
+  static args = [
+    {
+      name: 'name',
+      required: true,
+      description: 'A unique name to identify the node inside the cluster',
+    },
+  ]
+
+  static flags = {
+    cluster: Flags.string({
+      char: 'c',
+      required: true,
+      description: 'The name of the cluster where to add the node to',
+    }),
+    image: Flags.string({
+      char: 'i',
+      required: false,
+      description: 'The name of the Docker image to use to spawn the container',
```

Comment:
nit, can you add in the description, the default image is ironfish:latest
Like ironfish cli for datadir `-d, --datadir=<value>        [default: ~/.ironfish] The path to the data dir`